src/extractors/custom/otrs.com/index.js

```diff
@@ -0,0 +1,43 @@
+export const OtrsComExtractor = {
+  domain: 'otrs.com',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
```

Comment:
Let's fetch the title as it displays on the page from its actual heading tag since it's easily accessible, often the `og:title` is slightly different.
In this case, `#main article h1` should do.

Reply:
Thank you. Fixed.